kong/conf_loader.lua

```diff
@@ -294,6 +295,22 @@ local function check_and_infer(conf)
     end
   end
 
+  local mask = 0
+  if conf.tokens then
+    local allowed = constants.HEADER_MASKS
+    for _, token in ipairs(conf.tokens) do
+      if token == "off" then
+          mask = 0
+          break
+      elseif allowed[token] == nil then -- TODO what about case sensetive??
```

Comment:
style: need line jump before this line

Reply:
It's ok to be case-sensitive imho, and expect `server` and not `SERVER` or `ServER`.